diagnostic.md

```diff
@@ -5,6 +5,12 @@ GitHub repo.
 
 <!-- your answer starts here -->
 
+id
+class
+div
+::after
+
```

Comment:
I don't think you fully understand what is meant by "selector." Take a look at this article: http://www.w3schools.com/jquery/jquery_ref_selectors.asp